test/Microsoft.NET.Restore.Tests/GivenThatWeWantToUseFrameworkRoslyn.cs

```diff
@@ -46,6 +46,44 @@ public void It_restores_Microsoft_Net_Compilers_Toolset_Framework_when_requested
             }
         }
 
+        [Fact]
+        public void It_restores_Microsoft_Net_Compilers_Toolset_Framework_when_MSBuild_is_torn()
+        {
+            const string testProjectName = "NetCoreApp";
+            var project = new TestProject
+            {
+                Name = testProjectName,
+                TargetFrameworks = "net6.0",
+            };
+
+            // Add an explicit version for the test. This will normally come from the installer.
+            project.AdditionalProperties.Add("_NetFrameworkHostedCompilersVersion", "4.7.0-2.23260.7");
```

Comment:
Do we need this property?  Stage 0 should already have flowed from the installer so stage 2 should have this set I think.

Reply:
I was copying from the previous test in this class - I figured keeping them in sync was more relevant than testing the version-flow logic. Thoughts on that?